spec/services/pii/nist_encryption_spec.rb

```diff
@@ -24,6 +24,8 @@
     allow(FeatureManagement).to receive(:use_kms?).and_return(true)
   end
 
+  let(:kms_prefix) { '{}cH' } # XOR of 'KMSx'
```

Comment:
My first question when skimming this "XOR with what?". Should we specify it's `'KMSx' XOR '0000'`